compiler/rustc_builtin_macros/src/deriving/debug.rs

```diff
@@ -3,6 +3,7 @@ use crate::deriving::generic::*;
 use crate::deriving::path_std;
 
 use ast::EnumDef;
+use ast::MatchKind;
 use rustc_ast::{self as ast, MetaItem};
```

Comment:
```suggestion
use rustc_ast::{self as ast, EnumDef, MatchKind, MetaItem};
```
Editor import autoinsert.